frontend/model/contracts/group.js

```diff
@@ -622,7 +673,7 @@ sbp('chelonia/defineContract', {
       }
     },
     'gi.contracts/group/inviteAccept': {
-      validate: objectOf({
+      validate: objectMaybeOf({
```

Comment:
What is the reason for this change?

Reply:
I added some more fields and changed validation too.
But later, I reverted the code and I guess I didn't revert the validation.
I will revert it now. Nice catch.